mod.ts

```diff
@@ -61,12 +67,27 @@ const defaultOptions: LoadOptions = {
 export class Coffee {
   private loadOptions: LoadOptions = defaultOptions;
   private isLoaded = false;
+  private fileExts = [
+    "json",
+    "yml",
+  ];
 
   runtimeAPI: RuntimeAPI = new DenoAPI();
   parsers: { [k: string]: Parser } = {
     json: (t: string) => JSON.parse(t),
+    yml: (t: string) => this.ymlReader(t),
   };
   configs: Configs = {};
+  private ymlReader(t: string): Configs {
+    // ymlParse returns unknown, We should convert its type to Configs type
+    const parsedYml = ymlParse(t, {
+      schema: JSON_SCHEMA,
+      json: true,
+    });
+    const ymlToJSON = JSON.stringify(parsedYml);
```

Comment:
Why `stringify` an object to JSON then `parse` it? simply use 
```ts
return parsedYml as Configs
```

Reply:
ooooh, thanks erfan